TESTOWNERS

```diff
@@ -326,10 +324,12 @@
 # build_tests @eliasyishak @flutter/tool
 # ci_yaml flutter roller @caseyhillers @flutter/infra
 # coverage @godofredoc @flutter/infra
+# customer_testing @HansMuller @flutter/framework
```

Comment:
is this change effectively commenting out the customer_testing test owner logic? if so, why? if not, how does this work?

Reply:
Test owners validation is using "# <test name> <owner> <team>" for shard validations.